app/enterprise/2.1.x/plugins/grpc.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Introduction to Kong gRPC Plugins
+---
+
+Before we dive into the specifics of how to configure Kong's gRPC plugins, let's
+first discuss the advantages of the protocol. Unlike JSON, [gRPC](https://en.wikipedia.org/wiki/GRPC)
+is a binary protocol, using [protobuf](https://en.wikipedia.org/wiki/Protocol_Buffers)
+definitions to instruct how the data is marshalled and unmarshalled. Because
+binary data is used instead of text, it's a more efficient way to transmit data
+over a network. But this also makes gRPC harder to work with because inspecting
+what went wrong is more challenging. Additionally, few clients natively handle gRPC.
+
+To help alleviate the challenges of working with gRPC, Kong has two plugins:
+- [gRPC-Gateway](/hub/kong-inc/grpc-gateway)
+- [gRPC-Web](/hub/kong-inc/grpc-web)
+
+The gRPC-Gateway plugin allows you to send JSON requests to a gRPC service. A
+specially configured `.proto` file handles the conversion of the JSON request
+into one that the gRPC service can handle. This allows you to expose RESTful-style
+interfaces that talk to a gRPC service.
+
+The gRPC-Web plugin allows you to interact with a gRPC service from a browser.
+Instead of presenting a RESTful-type call, however, you POST data to the same
```

Comment:
```suggestion
Instead of presenting a RESTful-type call, you POST data to the same
```
Not sure the "however" really belongs here, up to you to rephrase otherwise if needed.